msteams-platform/task-modules-and-cards/cards/people-picker-adaptive-card.md

```diff
@@ -0,0 +1,246 @@
+            "text": "People Picker with Org search enabled"
+        },
+        {
+            "type": "Input.ChoiceSet",
+            "choices": [],
+            "choices.data": {
+                "type": "Data.Query",
+                "dataset": "graph.microsoft.com/users"
+            },
+            "id": "people-picker",
+            "isMultiSelect": true
+        },
+        {
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "size": "Medium",
+      "weight": "Bolder",
+      "text": "People Picker with Org search enabled"
+    },
+    {
+      "type": "Input.ChoiceSet",
+      "choices": [],
+      "choices.data": {
+        "type": "Data.Query",
+        "dataset": "graph.microsoft.com/users"
+      },
+      "id": "people-picker",
+      "isMultiSelect": true
+    }
+  ],
+  "actions": [
+    {
+      "type": "Action.Submit",
+      "title": "Submit"
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "version": "1.2"
+}
+```  
+
+The following image illustrates People Picker in Adaptive Cards with organization search:
+
+![People Picker Org Search](../../assets/images/cards/peoplepicker-org-search.png)
+
+To enable search within a list of conversation members, use the appropriate dataset defined in the [dataset](#dataset) table. `isMultiSelect` property is used to enable the selection of multiple users  in the control. It's set to false by default and this setting allows you to select single user only.
+
+### Data Submission
+
+You can use `Action.Submit` or `Action.Execute` to submit selected data to your bot. The `invoke` payload received on your bot is a list of AAD IDs or the IDs provided in static list.
+In People Picker, when a user is selected in the control, the `AAD ID` of the user is the value sent back. The `AAD ID` is a string and uniquely identifies a user in the directory.
+
+The format of the value submitted to the bot depends on the value of the `isMultiSelect` property:
+
+|value of `isMultiSelect`|Format|
+|--|--|
+|false _(single select)_|<selected_AAD_ID>.|
+|true _(multi select)_|<selected_AAD_ID_1>,<selected_AAD_ID_2>,<selected_AAD_ID_3>.|  
+
+With the `AAD ID`, People Picker preselects the corresponding user. 
+
+## Preselection of user
+
+People Picker supports preselection of user in the control, when creating and sending an Adaptive Card. `Input.ChoiceSet` supports the `value` property that is used to preselect a user. The format of this `value` property is the same as the submitted value format in [data submission](#data-submission).  
+The following list provides the information to preselect users:
+
+* For single user in the control, specify the `AAD ID` of the user as the `value`. 
+* For multiple users, such as `isMultiSelect` is `true`, specify a comma-separated string of `AAD ID`s.   
+
+The following image illustrates the preselection scenario:
+
+![Preselection Scenario](../../assets/images/cards/peoplepicker-preselection.png)     
```

Comment:
The card in this image is from a test bot and could be quite misleading for users. We should provide json snippets instead.

Reply:
we can use the following snippets as samples:
For pre-selecting a single person:
`{
      "type": "Input.ChoiceSet",
      "choices": [],
      "choices.data": {
        "type": "Data.Query",
        "dataset": "graph.microsoft.com/users"
      },
      "id": "people-picker",
      "value": "<AAD ID 1>"
    }`

For pre-selecting multiple people:
`{
      "type": "Input.ChoiceSet",
      "choices": [],
      "choices.data": {
        "type": "Data.Query",
        "dataset": "graph.microsoft.com/users"
      },
      "id": "people-picker",
      "isMultiSelect": true,
      "value": "<AAD ID 1>,<AAD ID 2>,<AAD ID 3>"
    }`

Reply:
Added